header/log_includes/debug.h

```diff
@@ -0,0 +1,6 @@
+
+#include<string>
+using namespace std;
```

Comment:
Don’t write namespace usings in a header file or before an #include.
Ref: http://www.gotw.ca/publications/c++cs.htm